src/App.js

```diff
@@ -1,55 +1,55 @@
 import React, { Component } from "react";
 import { BrowserRouter, Switch, Route } from "react-router-dom";
-import LandingView from "./modules/landing/Landing";
-import HomeScreen from "./modules/admin/Home/home";
-import ProfileScreen from "./modules/admin/Profile/Profile"; 
-import SalesScreen from "./modules/admin/Sales/Sales";
-import NotificationsScreen from "./modules/admin/Notifications/Notifications";
-import InventoryScreen from "./modules/admin/Invetory/Invetory";
-import { ToastContainer } from 'react-toastify';
-
-
+import LandingView from "./modules/landing/LandingView";
+import HomeView from "./modules/home/homeView";
```

Comment:
React components and Views are Always Camel Case

Reply:
No sera Pascal Case?

Reply:
si, me equivoque